app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
```

Comment:
Add a blank line between different import groups to follow PEP-8

Reply:
added a blank